react-components/src/architecture/base/concreteCommands/pointCloud/PointCloudDividerCommand.test.ts

```diff
@@ -0,0 +1,19 @@
+import { PointCloudDividerCommand } from './PointCloudDividerCommand';
+import { createFullRenderTargetMock } from '../../../../../tests/tests-utilities/fixtures/createFullRenderTargetMock';
+import { describe, expect, test } from 'vitest';
+import { PointCloudDomainObject } from '../../../concrete/reveal/pointCloud/PointCloudDomainObject';
+import { createPointCloudMock } from '../../../../../tests/tests-utilities/fixtures/pointCloud';
```

Comment:
should be imported as

```suggestion
import { createPointCloudMock } from '#test-utils/fixtures/pointCloud';
```